dcrec/secp256k1/curve_embedded.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-2024 The Decred developers
```

Comment:
This is a new file, so it should only be the current year and not attributed to btcsuite either.